backend/loadConfig.js

```diff
@@ -0,0 +1,55 @@
+const fs = require('fs');
```

Comment:
I would rename this file to `config.js`, because it load export config, not loading functionality.